packages/sdk/src/conversation/interface.ts

```diff
@@ -436,7 +436,7 @@ export interface ConversationOptions {
  *    ...
  * });
  * ```
- * For details information about how to implement a BotSsoExecutionActivityHandler, please refer DefaultBotSsoExecutionActivityHandler class source code.
+ * For details information about how to implement a BotSsoExecutionActivityHandler, please refer DefaultBotSsoExecutionActivityHandler class source code: https://github.com/OfficeDev/TeamsFx/blob/dev/packages/sdk/src/conversation/sso/defaultBotSsoExecutionActivityHandler.ts
```

Comment:
Use an aka link here in case the url is not valid any more?

Reply:
Sure, will update in another PR, and to make sure this PR can catch up today's release~